lambda.yaml

```diff
@@ -29,14 +29,15 @@
             dest: "{{ gxy_io_zip_path }}"
 
         - name: Deploy Lambda function
-          community.aws.lambda:
+          amazon.aws.lambda:
             name: "gxy-io-redirect"
             state: present
             zip_file: "{{ gxy_io_zip_path }}"
             runtime: "python3.9"
             role: "{{ gxy_io_lambda_exec_role_arn }}"
             handler: "lambda_function.lambda_handler"
             region: "us-east-1"
+          no_log: true
```

Comment:
This was a long shot, but it just hides the output from being printed -- it's still returned and fails the step.